### PR TITLE
conf: NOTICE() on mounts on container's /dev

### PR DESCRIPTION
**Pull request by brauner**
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

File: src/lxc/conf.c [hunk 1]
```diff
@@ -2016,14 +2016,56 @@ static int mount_entry_on_relative_rootfs(struct mntent *mntent,
 	return mount_entry_on_generic(mntent, path, rootfs, lxc_name, lxc_path);
 }
 
-static int mount_file_entries(const struct lxc_rootfs *rootfs, FILE *file,
+static void notice_on_conflict(const struct lxc_conf *conf, const char *src,
```
Comment (hallyn) [inline src/lxc/conf.c]: Please put a comment here explaining precisely what amounts to a conflict.

**Comment by hallyn** [comment]
(notice build error - ../../../src/lxc/conf.c  -fPIC -DPIC -o .libs/liblxc_la-conf.o
../../../src/lxc/conf.c: In function ‘mount_file_entries’:
../../../src/lxc/conf.c:2067:3: error: implicit declaration of function ‘warn_on_conflict’ [-Werror=implicit-function-declaration]
   warn_on_conflict(conf, mntent.mnt_fsname, mntent.mnt_dir);
   ^
../../../src/lxc/conf.c: At top level:
../../../src/lxc/conf.c:2019:13: error: ‘notice_on_conflict’ defined but not used [-Werror=unused-function]
 static void notice_on_conflict(const struct lxc_conf *conf, const char *src,
)

**Comment by brauner** [comment]
@hallyn, yeah, I'm on it. :)

**Comment by brauner** [comment]
@hallyn, updated

**Pull request merged**